app/assets/stylesheets/frontend/views/_html-publications.scss

```diff
@@ -286,6 +286,71 @@
           }
         }
       }
+
+      table.financial-data {
+
+        .numeric {
+          text-align: right;
+          font-family: $NTA-Light-Tabular;
+        }
+
+        // make all elements inside thead look the same
+        // (td can sometimes be used to overcome complexity of multiple lines in thead)
+        // and make all of thead and tfoot stand out
+        thead,
+        tfoot {
+          background-color: #e1e8e8;
+        }
+        thead th,
+        thead td {
+          font-weight: bold;
+        }
+        // don't make `th`s bold unless they are a section heading or a (sub)total row
+        tbody th,
+        tfoot th {
+          font-weight: normal;
+        }
+
+        // needs padding due to occasional background colour
+        // is intentionally small and not balanced with other padding
+        // as tables can be quite wide
+        tr > :first-child {
+          padding-left: 5px;
+        }
+
+        // add spacing so that groupings are clearer
+        tr.section-heading > * {
+          font-weight: bold;
+          padding-top: 30px;
```

Comment:
This can be `padding-top: $gutter` I think, as well as the other pixel measurements in this file except for the `border-top: 3px`.